apps/server/src/modules/tldraw/service/tldraw.ws.service.spec.ts

```diff
@@ -446,4 +461,25 @@ describe('TldrawWSService', () => {
 			flushDocumentSpy.mockRestore();
 		});
 	});
+
+	describe('authorizeConnection', () => {
+		it('should call properly method', async () => {
+			const params = { drawingName: 'drawingName', token: 'token' };
+			const response: AxiosResponse<null> = axiosResponseFactory.build({
+				status: 200,
+			});
+
+			httpService.get.mockReturnValueOnce(of(response));
+
+			await expect(service.authorizeConnection(params.drawingName, params.token)).resolves.not.toThrow();
+		});
```

Comment:
missing a test to check that the endpoint is called with given path and headers etc?